remotes/docker/config/hosts.go

```diff
@@ -108,9 +108,9 @@ func ConfigureHosts(ctx context.Context, options HostOptions) docker.RegistryHos
 		defaultTransport := &http.Transport{
 			Proxy: http.ProxyFromEnvironment,
 			DialContext: (&net.Dialer{
-				Timeout:   30 * time.Second,
-				KeepAlive: 30 * time.Second,
-				DualStack: true,
+				Timeout:       30 * time.Second,
+				KeepAlive:     30 * time.Second,
+				FallbackDelay: 300 * time.Millisecond,
```

Comment:
What about using default value? zero value means 300 ms

Reply:
I outlined some of my motivation on https://github.com/containerd/cri/pull/1475#discussion_r422656281. Happy to change this though (in which case I can remove the `FallbackDelay` here); I thought it would be more transparent/explicit to show what default we use (and not get surprised by possible changes to the default in golang)